doc/install/quickstart.rst

```diff
@@ -0,0 +1,52 @@
+.. include:: ../links.inc
+
+.. _quick-start:
+
+Quick start
+===========
+
+MNE-Python requires Python version |min_python_version| or higher. If you've
+never worked with Python before, skip ahead to the last paragraph of this page.
+For users already familiar with Python:
+
+- If you only need MNE-Python's computational functions, only hard dependencies
+  will be included when running:
+
+  .. code-block:: console
+
+      $ pip install mne
```

Comment:
I think somewhere in the documentation it should also be mentioned how to upgrade the MNE version. Is this done somewhere?

Reply:
also shouldn't you add ``--user`` to prevent folks from doing "sudo" specially if they are on a Anaconda package that is managed by the center.

Reply:
> I think somewhere in the documentation it should also be mentioned how to upgrade the MNE version. Is this done somewhere?

With the installers, you'd just install the new version and you're set.